Products/LDAPMultiPlugins/ActiveDirectoryMultiPlugin.py

```diff
@@ -165,7 +165,15 @@ def __init__(self, id, title='', groupid_attr='objectGUID',
         self.grouptitle_attr = grouptitle_attr
         self.group_class = group_class
         self.group_recurse = group_recurse
-        self.group_recurse_depth = group_recurse_depth
+
+    def get_lookup_groups_base(self, acl):
+        """Return lookup_groups_base if it exists otherwhise the group_base.
+        """
+        if hasattr(acl, 'lookup_groups_base'):
```

Comment:
In general I would not use `hasattr()` in Python 2 code as it can swallow exceptions.
In this case something like `getattr(acl, 'lookup_groups_base', acl.groups_base)` looks simpler to me.